releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -14,6 +14,130 @@
+* This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.8.
+
+## 5.6.0 - October 24, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.1.3.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.4.
+
+## Bugs Fix
+
+* Fixes an issue where Web Hook authorization was validated incorrectly, resulting in an Unauthorized error [#37141](https://github.com/Azure/azure-sdk-for-java/pull/37141).
+
+## 4.12.0 - October 23, 2023
+
+* This release is compatible with Spring Boot 2.5.0-2.5.15, 2.6.0-2.6.15, 2.7.0-2.7.16.
+* This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.8.
+
+## Bugs Fix
+
+* Fixes an issue where Web Hook authorization was validated incorrectly, resulting in an Unauthorized error [#37141](https://github.com/Azure/azure-sdk-for-java/pull/37141).
+
+## 5.5.0 - August 28, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.1.2.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.4.
+
+## 4.11.0 - August 25, 2023
+
+* This release is compatible with Spring Boot 2.5.0-2.5.15, 2.6.0-2.6.15, 2.7.0-2.7.14.
+* This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.8.
+
+## 5.4.0 - August 02, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.1.0.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.3.
+
+### Bugs Fixed
+
+* Fixes a bug where targeting exclusions don't map correctly resulting in a java.lang.ClassCastException [#35823](https://github.com/Azure/azure-sdk-for-java/issues/35823).
+
+## 4.10.0 - August 01, 2023
+
+* This release is compatible with Spring Boot 2.5.0-2.5.15, 2.6.0-2.6.15, 2.7.0-2.7.13.
+* This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.7.
+
+## 4.9.1 - July 19, 2023
+
+### Bugs Fixed
+
+* Fixes a bug where targeting exclusions don't map correctly resulting in a java.lang.ClassCastException [#35823](https://github.com/Azure/azure-sdk-for-java/issues/35823).
+
+## 5.3.0 - June 28, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.1.0.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.3.
+
+## 4.9.0 - June 29, 2023
+
+* This release is compatible with Spring Boot 2.5.0-2.5.15, 2.6.0-2.6.15, 2.7.0-2.7.11.
+* This release is compatible with Spring Cloud 2020.0.3-2020.0.6, 2021.0.0-2021.0.7.
+
+## 5.2.0 - June 01, 2023
+
+* This release is compatible with Spring Boot 3.0.0-3.0.5.
+* This release is compatible with Spring Cloud 2022.0.0-2022.0.2.
+
+### Bugs Fixed
+
+* Fixes issue where credential from Azure Spring global properties was being overridden [#34694](https://github.com/Azure/azure-sdk-for-java/pull/34694).
+* Fixes bug where Http Response wasn't checked before trying to use response [#35086](https://github.com/Azure/azure-sdk-for-java/pull/35086).
+* Fixes Tracing info for ContainerApp [#35086](https://github.com/Azure/azure-sdk-for-java/pull/35086).
```

Comment:
Telemetry changes don't need to be included in release notes. Also, the same PR is already linked above, so there's not really any new information we are providing here.